src/shared/scrapers/USA/CO/index.js

```diff
@@ -141,6 +141,7 @@ const scraper = {
       }
       const stateData = transform.sumData(counties);
       stateData.population = parse.number(data[0].State_Population);
+      stateData.deaths = parse.number(data[0].State_Deaths);
```

Comment:
Does this work on older cached versions? Has the State_Deaths field been these since we started scraping it?

Reply:
Yes, the data was always there
Also realized I was double wrong about `datetime.scrapeDateIsBefore()`. It should be `datetime.scrapeDateIsAfter()`